packages/react/src/components/DataTableSkeleton/DataTableSkeleton.tsx

```diff
@@ -55,6 +56,9 @@ export interface DataTableSkeletonProps
    */
   showToolbar?: boolean;
 
+  
+  size?: DataTableSize;
```

Comment:
Has this code been formatted?


```suggestion
  /*
   * Changes the row height of table.
   */
  size?: DataTableSize;
```